### DISCUSSION TITLE
New device awning

### DISCUSSION THREAD

**Discussion by sebcbien37** [HomeSpan/HomeSpan#603] [Ideas]
Hello,

Thank you for your work !

Is it possible to add an Awning ? 

Thank.
Have a nice day.

Sébastien from France.

**Comment by HomeSpan** [comment]
The Apple HomeKit Categories and Services are somewhat generic and can be used for anything where the controls in HomeKit make sense for a particular device.  I would recommend using the Window Covering Service to control an awning since Siri will recognize the words "open" and "close" for this type of device.

**Reply by sebcbien37** [reply]
Thank you for your reply.
I would have liked to have an icon of an Awning. It's not very important, thank you anyway!
![image](https://github.com/HomeSpan/HomeSpan/assets/73611880/22456c78-4441-4dff-9ecd-9a1452b8fdf2)

**Comment by HomeSpan** [comment]
The Home App contains a few different icons for Window Shades, but nothing that really looks like an awning.   It's too bad that the Home App is not customizable - would be ideal if a device can transmit it's own icon during set-up.

**Reply by sebcbien37** [reply]
Can you tell me where I can change the icon used for each device? I know a little about programming, I want to see which icon I would like the most!

**Comment by HomeSpan** [comment]
It's in the Home App on the iPhone, available for all HomeKit devices (nothing to do with HomeSpan itself).  Click on a Home App tile, then click the settings icon, then click on the icon itself to edit.  There are limited choices based on the type of device.
